<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logicWidget</class>
 <widget class="QWidget" name="logicWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Объект</string>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="typeLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип</string>
    </property>
   </widget>
   <widget class="QLabel" name="addressLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Адрес:</string>
    </property>
   </widget>
  </widget>
  <widget class="portTableWidget" name="portWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="onTableWidget" name="onWidget" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="offTableWidget" name="offWidget" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>portTableWidget</class>
   <extends>QWidget</extends>
   <header location="global">widgets/porttablewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>onTableWidget</class>
   <extends>QWidget</extends>
   <header location="global">widgets/ontablewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>offTableWidget</class>
   <extends>QWidget</extends>
   <header location="global">widgets/offtablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
